Expense report: add different currency to choose from

ffit2020 opened the pull request:
# Instructions
*This is a template to help you make good pull requests. You may use [Github Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/) syntax to format your issue report.*
*Please:*
- *only keep the "Fix", "Close" or "New" section*
- *follow the project [contributing guidelines](/.github/CONTRIBUTING.md)*
- *replace the bracket enclosed textswith meaningful informations*


# Fix #[*issue_number Short description*]



# Close #[*issue_number Short description*]
[*Long description*]


# New [Expense report: add different currency to choose from]
It could be great to have a dropdown in expense report, where users can choose from different currencies than the default one

atm-maxime commented:
Hey ! I think you're confused between issue and pull request. If you want to open a discussion about a new feature, you have to create an issue, not a PR. Besides, your PR is to merge develop into 13.0, which is wrong, we never do that, it's the other way around.

eldy committed:
Complete information on tab for files on salary

eldy committed:
Debug module salary

eldy committed:
Debug v14

Commit:
New check update for modules

eldy committed:
css

eldy committed:
Easier to click

eldy committed:
Position autoset

eldy committed:
Clean code

eldy committed:
css

Commit:
New check update for modules

Commit:
remove space

stickler-ci committed:
Fixing style errors.

Commit:
Fix code Stickler

stickler-ci committed:
Fixing style errors.

Commit:
Remove ctrl+v

Commit:
Merge branch 'new/check_update_for_module' of github.com:atm-john/dolibarr into new/check_update_for_module

eldy committed:
Save default langueg and other language in export of website

eldy committed:
css

eldy committed:
Accept several sizes for favicon

eldy committed:
Bad test

eldy committed:
Add a schedule dtask to send reminder by email on date due.

eldy committed:
Add a schedule dtask to send reminder by email on date due.

eldy committed:
Fix id cron task

eldy committed:
Load of template before the loop.

prietojc committed:
Update bonprelevement.class.php

Change facture reference by supplier reference

eldy committed:
Can filter also on payment mode

eldy committed:
Fix pb in date format %a %b when using a negative timezone
Enhance date management and unit tests.

fr69400 committed:
Update contact.php

eldy committed:
Add local ip into excluded IP for external URL download.
Fix #yogosha5861

eldy committed:
Typo

eldy committed:
Fix message

eldy committed:
Fix update pass

eldy committed:
Clean file

dolibit-ut committed:
Update agenda.php

$wikihelp  =>  $help_url

eldy committed:
Trans

eldy committed:
Try to use 2 different names for operation

eldy committed:
Better error message

eldy committed:
NEW Add option MAIN_SECURITY_ANTI_SSRF_SERVER_IP

eldy committed:
Check file name end with .xml

eldy committed:
Fix error message

eldy committed:
Use MAIN_SECURITY_ANTI_SSRF_SERVER_IP as whitelist for local IP.

dolibit-ut committed:
Update index.php

add comments
// init API
// init PHPUnit
// init ExtraFields
// init Hook
// init Trigger
// init Widget
// init CSS
// init JS
// init CLI
// init Doc
// add Language
// remove/delete File

stickler-ci committed:
Fixing style errors.

eldy committed:
Show MAIN_SECURITY_ANTI_SSRF_SERVER_IP option into security page

eldy committed:
Fix escape

eldy committed:
Fix phpcs

dolibit-ut committed:
Update llx_10_c_regions.sql

-- Mauritius
-- Venezuela
-- United Arab Emirates
-- Portugal
-- Slovenia (correction fk_pays)

eldy committed:
Fix perm

dolibit-ut committed:
Update document.php

$helpurl  =>  $help_url 

$help_url = "EN:Module_Expense_Reports|FR:Module_Notes_de_frais";

dolibit-ut committed:
Update index.php

$help_url = "EN:Module_Expense_Reports|FR:Module_Notes_de_frais";

stickler-ci committed:
Fixing style errors.

dolibit-ut committed:
Update list.php

$xxxx_start / $xxxx_end :   " "  =>  ' '

dolibit-ut committed:
Update mo_card.php

$title = $langs->trans('Mo')." - ".$langs->trans("Card");

dolibit-ut committed:
Update mo_movements.php

$help_url = 'EN:Module_Manufacturing_Orders|FR:Module_Ordres_de_Fabrication';

dolibit-ut committed:
Update index.php

$title = $langs->trans('HRMArea');

llxHeader('', $title, '');

dolibit-ut committed:
Update admin_establishment.php

$title = $langs->trans('Establishments');

llxHeader('', $title, '');

dolibit-ut committed:
Update admin_hrm.php

$title = $langs->trans('Parameters');

llxHeader('', $title, '');

dolibit-ut committed:
Update comment.php

$title = $langs->trans('CommentPage');

llxHeader('', $title, '');

dolibit-ut committed:
Update contact.php

$title / $help_url    " "  =>  ' '
add   DE:Modul_Projekte

stickler-ci committed:
Fixing style errors.

dolibit-ut committed:
Update document.php

$title / $help_url " " => ' '
add  DE:Modul_Projekte

stickler-ci committed:
Fixing style errors.

dolibit-ut committed:
Update element.php

$title / $help_url / llxHeader   " " => ' '
add  DE:Modul_Projekte
llxHeader('', $title, $help_url);

stickler-ci committed:
Fixing style errors.

dolibit-ut committed:
Update index.php

new:  $help_url
add  DE:Modul_Projekte
" " => ' '
llxHeader('', $title, $help_url);

eldy committed:
Merge pull request #17340 from dolibit-ut/patch-295

Update index.php

eldy committed:
Merge pull request #17338 from dolibit-ut/patch-294

Update element.php

eldy committed:
Merge pull request #17337 from dolibit-ut/patch-293

Update document.php

eldy committed:
Merge pull request #17336 from dolibit-ut/patch-292

Update contact.php

eldy committed:
Merge pull request #17334 from dolibit-ut/patch-290

Update admin_hrm.php

eldy committed:
Merge pull request #17332 from dolibit-ut/patch-288

Update index.php

eldy committed:
Merge pull request #17333 from dolibit-ut/patch-289

Update admin_establishment.php

eldy committed:
Merge pull request #17330 from dolibit-ut/patch-286

Update mo_card.php

eldy committed:
Update mo_movements.php

eldy committed:
Merge pull request #17331 from dolibit-ut/patch-287

Update mo_movements.php

eldy committed:
Merge pull request #17335 from dolibit-ut/patch-291

Update comment.php

eldy committed:
Merge pull request #17329 from dolibit-ut/patch-285

Update list.php

eldy committed:
Merge pull request #17325 from dolibit-ut/patch-282

Update llx_10_c_regions.sql

eldy committed:
Fix phpunit

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge pull request #17324 from dolibit-ut/patch-281

Update index.php

eldy committed:
Merge pull request #17327 from dolibit-ut/patch-283

Update document.php

eldy committed:
Merge pull request #17328 from dolibit-ut/patch-284

Update index.php

eldy committed:
Avoid error on PHP 5.6

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge branch '13.0' of git@github.com:Dolibarr/dolibarr.git into develop

Conflicts:
	htdocs/comm/mailing/cibles.php
	htdocs/core/extrafieldsinexport.inc.php
	htdocs/product/stock/card.php

eldy committed:
Merge pull request #17320 from dolibit-ut/patch-279

Update agenda.php

eldy committed:
Merge pull request #17315 from fr69400/develop

Update supplier_proposal\contact.php

eldy committed:
Update bonprelevement.class.php

eldy committed:
Add invoice id into EndToEnd to guarantee unicity of value.

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge pull request #17312 from prietojc/patch-16

Update bonprelevement.class.php

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge branch 'develop' into new/check_update_for_module

eldy committed:
Merge pull request #17309 from atm-john/new/check_update_for_module

NEW : Check update for externals modules using button on module page

eldy committed:
Fix HTML5

eldy committed:
Merge pull request #17297 from dolibit-ut/patch-278

Update llx_20_c_departements.sql

eldy committed:
Merge pull request #17303 from daraelmin/daraelmin-patch-newmemberform

Fix new member form use getpost instead FORCEXXX

eldy committed:
Fix ident

eldy committed:
Merge branch 'develop' into daraelmin-patch-2

eldy committed:
Merge pull request #17305 from daraelmin/daraelmin-patch-2

Fix warning wrong date format for new member form

eldy committed:
Merge pull request #17279 from aspangaro/14a38

NEW: Add constant MAIN_BUGTRACK_URL to add possibility to redirect to other service

eldy committed:
Merge pull request #17255 from Givriz/dev_1

Compatibility phpv8

eldy committed:
Merge branch 'develop' into fix_rebuild_sequence_for_postgres

eldy committed:
Merge pull request #17276 from atm-gauthier/fix_rebuild_sequence_for_postgres

FIX : Rebuild sequence for postgres only after query "INSERT INTO llx_salary(rowid..."

eldy committed:
Merge pull request #17302 from aspangaro/14a40

Facture_fourn - Field tva is deprecated

eldy committed:
Merge pull request #17275 from DorianVabre/dev_EventOrganizer

Dev event organizer: changes on the fields required on the conference attendee subscription page

eldy committed:
Merge pull request #17263 from atm-adrien/NEW_add_userGroup_for_user_exports_develop

NEW : Add possibility to exports userGroups fields on user exports

eldy committed:
Merge pull request #17248 from altairisfr/batch_mask_product

Batch mask product

eldy committed:
css

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Sync transifex

altairis-noe committed:
product card typo

dolibit-ut committed:
Update element.php

merging by
- if (!empty($conf->facture->enabled))
- if (!empty($conf->loan->enabled))

dolibit-ut committed:
Update bom_agenda.php

$help_url = 'EN:Module_Agenda_En|FR:Module_Agenda|ES:Módulo_Agenda';

dolibit-ut committed:
Update bom_card.php

$help_url ='EN:Module_BOM';
llxHeader('', $title, $help_url);

dolibit-ut committed:
Update bom_document.php

$help_url = 'EN:Module_BOM';

dolibit-ut committed:
Update bom_list.php

new: $help_url = 'EN:Module_BOM';

moved up:  llxHeader('', $title, $help_url);

eldy committed:
Add the setup page of module accounting to not search where is setup

eldy committed:
Enhance detection of update available

eldy committed:
Fix trans

eldy committed:
Fix CURLOPT_CONNECT_TO

eldy committed:
css

eldy committed:
Fix php8 warning

dolibit-ut committed:
Update llx_20_c_departements.sql

Italy new

eldy committed:
FIX pb when saving some export profiles

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

daraelmin committed:
Update paymentok.php

Commit:
Merge remote-tracking branch 'origin/13.0' into 13-company-disable-workflow

stickler-ci committed:
Fixing style errors.

Robin-Yann committed:
Merge branch 'develop' into 13-company-disable-workflow

stickler-ci committed:
Fixing style errors.

daraelmin committed:
Missing $

dolibit-ut committed:
Update llx_10_c_regions.sql

Peru
Hungary

indelog committed:
Add : register donation payment when public payment

- When we paid a donation whit public link, create the associated payment and
  put its status to payed if all the promised amount is it.

- Set the amount to pay on public page depending to the rest to pay for
  the don.

cfoellmann committed:
fix bookmarks to allow Public items to be set and deleted by admins

stickler-ci committed:
Fixing style errors.

indelog committed:
Fix Stickler

Commit:
FIX tag selector error if supplier disable

Robin-Yann committed:
Merge branch 'develop' into 13-fix-contact_filer-fournisseur

stickler-ci committed:
Fixing style errors.

indelog committed:
Fix : missig ext_payment_id on adh public pay

When pay a mebership contibution whit public link the payment add add in
database don't have a "ext_payment_id". This fix it.

gmilad committed:
Update doleditor.class.php for 

(English below)
Bonjour,

Voici une proposition d'ajout de fonctionnalité afin de pouvoir activer facilement le plugins SpellCheckAsYouType (SCAYT)  de CKEditor, pour l'historique voir ce fils de discutions sur le forum Francophone de Dolibarr :

https://www.dolibarr.fr/forum/t/wysiwyg/28874/11?u=gmilad

Amicalement,
Gaëtan.

(English below)
Hello,

Here is a proposal to add functionality to easily activate the  SpellCheckAsYouType (SCAYT)  plugins of CKEditor, for the history see this thread on the Dolibarr forum:

https://www.dolibarr.fr/forum/t/wysiwyg/28874/11?u=gmilad

Kind regards,
Gaëtan.

ibuiv committed:
Product Stock Card - add new standard Hooks

stickler-ci committed:
Fixing style errors.

aspangaro committed:
NEW: Add option SUPPLIER_PROPOSAL_ALLOW_EXTERNAL_DOWNLOAD

eldy committed:
Fix protect sql

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Fix #yogosha5877

eldy committed:
Fix #yogosha5877

eldy committed:
Fix #yogosha5877

eldy committed:
FIX #yogosha5879

eldy committed:
Clean edit of user card

eldy committed:
css

eldy committed:
Fix #yogosha5881

eldy committed:
Typo

eldy committed:
Upgrade to swagger 2.2

eldy committed:
Clean API

eldy committed:
Escape string

eldy committed:
Fix escape

eldy committed:
Allow option to restrict API endpoints

stickler-ci committed:
Fixing style errors.

eldy committed:
Fix phpcs

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Fix web site

eldy committed:
Fix test on Local url

cfoellmann committed:
fix missing backets

Commit:
FIX combo Prospect / Customers / Supplier in society list

Robin-Yann committed:
Merge branch 'develop' into 13-fix-combo-prospect

stickler-ci committed:
Fixing style errors.

Robin-Yann committed:
Update card.php

eldy committed:
Clean code

atm-gauthier committed:
FIX : We must not be able to delete sociale charge, vat or salary if there are payments

atm-gauthier committed:
FIX : useless column

eldy committed:
Clean code

atm-gauthier committed:
FIX : last modif date for salary

Robin-Yann committed:
Update card.php

Robin-Yann committed:
Update card.php

stickler-ci committed:
Fixing style errors.

eldy committed:
Clean code

stickler-ci committed:
Fixing style errors.

Commit:
FIX Supplier invoices (deposit, multicurency in lines)

Commit:
Fix Stickler CI

eldy committed:
Clean code

eldy committed:
Clean code

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
fix css

eldy committed:
Fix amount chosen lost

eldy committed:
Merge pull request #17396 from atm-gauthier/fix_disable_delete_if_payments

FIX : We must not be able to delete sociale charge, vat or salary if there are payments

eldy committed:
Merge pull request #17388 from aspangaro/14a30

NEW: Add option SUPPLIER_PROPOSAL_ALLOW_EXTERNAL_DOWNLOAD

eldy committed:
Merge pull request #17375 from indelog/fix_adhpayextid

Fix : missig ext_payment_id on adh public pay

eldy committed:
Merge pull request #17384 from ibuiv/patch-5

Product Stock Card - add new standard Hooks

eldy committed:
Update list.php

eldy committed:
Merge pull request #17374 from Robin-Yann/13-fix-contact_filer-fournisseur

FIX List contact TAG filter fournisseur

eldy committed:
Merge pull request #17373 from indelog/public_donationpaiement

Add : register donation payment when public payment

eldy committed:
Merge pull request #17377 from gmilad/gmilad-scayt-210423

NEW : Update doleditor.class.php for easily activate SCAYT

eldy committed:
Merge pull request #17367 from dolibit-ut/patch-302

Update llx_10_c_regions.sql

eldy committed:
Fix id in comment

eldy committed:
Merge pull request #17370 from cfoellmann/fix-bookmarks

NEW Make public bookmarks editable by admin users only

eldy committed:
Fix regression

eldy committed:
Look and feel v14

eldy committed:
Add method checkValForAPI to clean data from API input

eldy committed:
Clean code

eldy committed:
Clean code

eldy committed:
Update card.php

eldy committed:
Merge pull request #17395 from Robin-Yann/13-fix-combo-prospect

13 fix combo prospect

eldy committed:
Merge pull request #17346 from dolibit-ut/patch-296

Update element.php

eldy committed:
Merge pull request #17364 from Robin-Yann/13-company-disable-workflow

NEW Add option SOCIETE_DISABLE_WORKFORCE to hide staff field

eldy committed:
Merge pull request #17353 from dolibit-ut/patch-301

Update llx_20_c_departements.sql

eldy committed:
Merge pull request #17344 from altairis-noe/product_card_typo

product card typo

eldy committed:
Merge pull request #17347 from dolibit-ut/patch-297

Update bom_agenda.php

eldy committed:
Merge pull request #17348 from dolibit-ut/patch-298

Update bom_card.php

eldy committed:
Merge pull request #17349 from dolibit-ut/patch-299

Update bom_document.php

eldy committed:
Merge pull request #17350 from dolibit-ut/patch-300

Update bom_list.php

daraelmin committed:
$object::STATUS_RESILIATE

eldy committed:
Merge branch '13.0' of git@github.com:Dolibarr/dolibarr.git into develop

Conflicts:
	htdocs/compta/paiement/card.php
	htdocs/compta/paiement/class/paiement.class.php
	htdocs/public/members/new.php

eldy committed:
Merge branch '13.0' of git@github.com:Dolibarr/dolibarr.git into develop

Conflicts:
	htdocs/compta/paiement/card.php

eldy committed:
Merge branch '13.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Package for v14

daraelmin committed:
Draft and resiliated member could (re)validate

eldy committed:
Enhance ticket module

eldy committed:
Merge branch '13.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Fix mass action delete of cron jobs

eldy committed:
Enhance logs

eldy committed:
Show name of sitemap file

eldy committed:
Fix replacement of sitemap entry in robot.txt

eldy committed:
Fix alternate into sitemap

Commit:
Corrections

eldy committed:
FIX Disabled pages must not appears into sitemap

Commit:
Merge branch 'develop' of github.com:Dolibarr/dolibarr into fix_00007

Commit:
Corrections

fr69400 committed:
Update supplier_proposal.class.php

stickler-ci committed:
Fixing style errors.

Commit:
Fix : Bug facture.class.php

eldy committed:
css

eldy committed:
Merge pull request #17410 from Hystepik/develop#3

Fix : Bug facture.class.php

eldy committed:
Fix phpcs

Commit:
Corrections

fr69400 committed:
Update supplier_proposal.class.php

eldy committed:
Merge pull request #17409 from fr69400/develop

Update supplier_proposal.class.php

eldy committed:
Merge pull request #17400 from OPEN-DSI/fix_00007

FIX Supplier invoices (deposit, multicurrency in lines)

eldy committed:
Make option label easier to understand.

eldy committed:
Add missing constants

eldy committed:
Update paymentok.php

daraelmin committed:
fix infinite loop

eldy committed:
Merge pull request #17366 from daraelmin/daraelmin-patch-laterenew

Fix #17318 false renew date

eldy committed:
FIX notifications for expense report has disappeared.

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Debug v14

eldy committed:
Debug v14
